src/components/Header/index.js

```diff
@@ -38,8 +68,14 @@ class Header extends Component {
       gameGuideType,
       gameGuideURL,
       tokenAddress,
+      mainnetAddress,
     } = this.props
 
+    let walletConnected = hasWallet
+    if (isTournament && shouldUseMetamask()) {
+      walletConnected = hasWallet && mainnetAddress
+    }
+
     const logoVars = {}
     if (isTournament) {
       logoVars['--logoAnnotation'] = "'Powered by Gnosis'"
```

Comment:
I know this is unrelated to the PR but can you explain this please? Why the double hyphen?

Reply:
CSS vars start with double hyphen. I did it with CSS variables because JS cannot do `import X from variable` or `const X = require(variable)` and we have to use different images. Maybe this could be done in other way - import multiple IMGs and then conditionally use one of them. What do you think?

Reply:
@andre-meyer